templates/etcd-ha/3/docker-compose.yml

```diff
@@ -1,41 +1,22 @@
 etcd:
-  image: rancher/etcd:v2.3.6
+  image: rancher/etcd:v2.3.6-4
   labels:
     io.rancher.scheduler.affinity:container_label_ne: io.rancher.stack_service.name=$${stack_name}/$${service_name}
     io.rancher.sidekicks: data
-    io.rancher.service.allocate.skip.serialize: 'true'
-    io.rancher.container.start_once: 'true'
   environment:
     ETCD_DATA_DIR: /data
     ETCDCTL_ENDPOINT: http://etcd:2379
+    RANCHER_DEBUG: ${DEBUG}
   links:
   - data
-  - discovery
   volumes_from:
-    - data
-# containerize data volume to enable restarts and upgrades
+  - data
+
 data:
   image: busybox
-  command: /bin/true
+  entrypoint: /bin/true
```

Comment:
My question wasn't around the usage of entrypoint vs command... 

My concern is for anyone that has deployed and attempts to upgrade.. we would delete the data volume because the container definition changed and therefore needs to be upgraded.